contracts/implementation/PoolFactory.sol

```diff
@@ -34,6 +35,11 @@ contract PoolFactory is IPoolFactory, ITwoStepGovernance {
     // Percent of fees that go to secondary fee address if applicable.
     uint256 public secondaryFeeSplitPercent = 10;
 
+    // Deployment fee variables
+    address public deploymentFeeToken;
+    uint256 public deploymentFee;
```

Comment:
Nitpick: Using a `uint32` or `uint16` here can save gas, because of storage packing

Reply:
I don't think we can pack this unfortunately, because the max integer size for `uint32` is ~ 10^9 and ERC20s use 18 decimal places so we need this value to accept > 10^18